src/sources/http_client/client.rs

```diff
@@ -116,7 +116,7 @@ const fn default_http_method() -> HttpMethod {
 fn query_examples() -> HashMap<String, Vec<String>> {
     HashMap::<_, _>::from_iter(
         [
-            ("field".to_owned(), vec!["value".to_owned()]),
```

Comment:
Was this just a test to trigger changes? Should it be kept in?

Reply:
👋 this is a pr in the plaground repo :)

Reply:
😂 Didn't even notice

Reply:
the good news is that im going to make a real PR for it shortly